README.md

```diff
@@ -26,17 +26,14 @@ The LeaderBoard App is a program that shows a list of score
 - Use async and await JavaScript features to consume the API. 
 
 
-<!-- ### 3. Adding Functionalites
-
-- I remove all hardcoded items from the tasks `array`.
-- I created a new JavaScript file for the new functionality.
-- Implemented a function for adding a new task `(add a new element to the array)`.
-- Implemented a function for editing task descriptions.
-- Implemented a function for deleting a task `(remove an element from the array)`.
-- Implemented a function for the "Clear all completed" button `(use filter() method)`.
-- By default new tasks have the property completed set to false and the property index set to the value of the new` array length`.
-- Deleting a task updates all remaining items' indexes, so they represent the current list order and are unique.
-- All changes to the To Do List are saved in `local storage`. -->
+### 3. final Touches
+
+- Now you should improve the look and feel of the application, adding the styles of your choice.  
+
+- Please keep the general layout of the wireframe, this is the only mandatory requirement. 
+
+- You can use plain CSS or any CSS framework, it's up to you. 
+
 
 ## Built With
```

Comment:
- Update your link for the live version:
[outdated link](https://alexander16108.github.io/LeaderBoard-API/dist/)

local:
<img width="1440" alt="Screen Shot 2021-10-14 at 6 47 06" src="https://user-images.githubusercontent.com/22258274/137311500-cf218b03-bb1a-4104-aa38-e0df910cc869.png">